generate_refresh_token.py

```
@@ -0,0 +1,60 @@
+import os
+import json
+from google.auth.transport.requests import Request
```

Ruff (0.8.2)

`google.auth.transport.requests.Request` imported but unused

Remove unused import: `google.auth.transport.requests.Request`

(F401)